fluentlenium-core/src/main/java/org/fluentlenium/core/domain/FluentList.java

```diff
@@ -259,6 +260,16 @@ public String getText() {
         return null;
     }
 
+    @Override
+    public FluentList<FluentWebElement> find(String name) {
+        return find(name, new Filter[0]);
+    }
+
+    @Override
+    public FluentList<FluentWebElement> find(String name, Predicate<FluentWebElement>... predicates) {
+        return null;
```

Comment:
I don't know what doesn't make sense, it's just incomplete.
Anyway, it doesn't really make sense to do the filtering in fluentlenium find when we can do it in assertj, I'll abort this PR.